presto-main/src/test/java/com/facebook/presto/execution/TestQueryInfo.java

```diff
@@ -201,6 +201,7 @@ private static QueryInfo createQueryInfo()
                                 new ValuesNode(Optional.empty(), new PlanNodeId("0"), ImmutableList.of(), ImmutableList.of(), Optional.empty()),
                                 PlanCanonicalizationStrategy.DEFAULT),
                         new PlanNodeCanonicalInfo("a", ImmutableList.of()))),
-                ImmutableMap.of());
+                ImmutableMap.of(),
+                Optional.empty());
```

Comment:
Should we enhance this test to add the `PrestoSparkExecutionContext` to prove it doesn't have issues serializing/deserializing to JSON?